src/app/auth/auth.component.html

```diff
@@ -39,11 +39,7 @@
     </div>
 
     <div *ngIf="!isUserAuthenticating()">
-      <button mat-stroked-button class="sign-in-button" color="primary" (click)="completeLoginProcess(currentUserName)"
-              [disabled]="isUserAuthenticating()">
-        <span class="logo"> <img class="github-logo" src="./assets/images/github-logo.png" alt="github-logo"> </span>
-        <span> Continue as {{this.currentUserName}} </span>
-      </button>
+      <app-auth-confirm-login [username]="currentUserName" [currentSessionOrg]="this.currentSessionOrg"></app-auth-confirm-login>
 
       <button *ngIf="electronService.isElectron()" mat-button style="margin-top: 10px; margin-bottom: 10px;" [disabled]="isUserAuthenticating()"
```

Comment:
I think we should move this part of the template too? Since it is part of the "confirmation process".

Reply:
@anubh-v 
Just to confirm, do you mean this whole part?
```html
<button *ngIf="electronService.isElectron()" mat-button style="margin-top: 10px; margin-bottom: 10px;" [disabled]="isUserAuthenticating()" (click)="logIntoAnotherAccount()"> Log in to another account </button>
<div *ngIf="!electronService.isElectron()" class="mat-body-1" style="margin-top: 20px;">
    To change account, please sign into the desired account from the
    <a href="https://github.com/" (click)="onGithubWebsiteClicked()">official Github website</a>.
</div>
```

After thinking for a while, yeah it seems to be part of the "confirmation process", since if the user don't want to proceed with the current account, he/she will need to change account.
Will make the changes, thanks for the comments!